lib/commentOnClosedIssue.js

```diff
@@ -45,11 +45,14 @@ commentOnClosedIssue._implementation = function (issueUrl, commentsUrl, reposito
     var comments = [];
     var issueHtmlUrl;
 
-    return requestPromise.get({
-        url: issueUrl,
-        headers: repositorySettings.headers,
-        json: true
-    })
+    return repositorySettings.fetchSettings()
```

Comment:
Doesn't have to be this PR, but a long term optimization could be to only fetch the files needed for a specific task rather than always fetching all of them. (Not a big deal for now though)